sponsors/pluralsight.md

```diff
@@ -0,0 +1,14 @@
+name: PluralSight
+tier: bronze
+website: https://www.pluralsight.com/
+twitter_handle: 
```

Comment:
1. Looking at the Pluralsight website, they don’t intercap the name – “Pluralsight”, not “PluralSight”.
2. They have a Twitter account: https://twitter.com/pluralsight